u-boot/pkg.yaml

```diff
@@ -12,7 +12,7 @@ variant: scratch
 shell: /toolchain/bin/bash
 dependencies:
   - stage: base
-  - stage: libressl
+  - stage: openssl
```

Comment:
does this change affect this config: https://github.com/talos-systems/pkgs/blob/37e757d2919af029df784ff9ab5ad2fe8eb30e8a/u-boot/pkg.yaml#L67